docs/topic-guides/checkpoints.txt

```diff
@@ -79,12 +79,18 @@ A checkpoint with UUID ``b3ed462c-a6c9-41e9-9202-5cb8ff00e109`` can be
 found in the directory
 ``/mnt/nfs-volume-1/b3ed462c-a6c9-41e9-9202-5cb8ff00e109``.
 
-The Determined CLI can be used to download a checkpoint saved on S3 or GCS:
+The Determined CLI can be used to download a checkpoint saved on Shared file system, S3 or GCS:
```

Comment:
non-blocking: I think this should read `a shared file system, S3 or GCS` since shared FS isn't the name of a specific product.

Reply:
done.